config/opal_check_cuda.m4

```diff
@@ -59,8 +59,8 @@ AC_ARG_WITH([cuda-libdir],
                             [Search for CUDA libraries in DIR])],
             [],
             [AS_IF([test -d "$with_cuda"],
-             [with_cuda_libdir=$(dirname $(find $with_cuda -name libcuda.so) 2> /dev/null)],
-             [with_cuda_libdir=$(dirname $(find /usr/local/cuda -name libcuda.so) 2> /dev/null)])
+             [with_cuda_libdir=$(dirname $(find $with_cuda -name libcuda.so 2> /dev/null) 2> /dev/null)],
```

Comment:
If you're going to change code on this PR, can you either split this into a 2nd commit or update the commit message to reflect what is actually contained in the commit?  The commit message currently only talks about updating the docs, but the commit now contains code changes, too.